ContractExamples/Cargo.toml

```diff
@@ -23,3 +28,9 @@ lto = true
 [profile.release-with-logs]
 inherits = "release"
 debug-assertions = true
+
+[profile.release.package.soroban-env-host]
```

Comment:
I don't see us using these additional profiles anywhere. 

Are they somehow expected?

Reply:
I believe it was also Stella